ReSharper.FSharp/src/FSharp.Psi/src/Impl/DeclaredElement/CompilerGenerated/FSharpPropertyAccessorMethod.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
+using FSharp.Compiler.SourceCodeServices;
 using JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Cache2.Parts;
 using JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Pointers;
+using JetBrains.ReSharper.Plugins.FSharp.Psi.Tree;
 using JetBrains.ReSharper.Psi;
 using JetBrains.ReSharper.Psi.Pointers;
 
 namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.DeclaredElement.CompilerGenerated
 {
-  internal class FSharpPropertyAccessorMethod : FSharpGeneratedMethodBase, IFSharpGeneratedAccessor
+  internal class FSharpPropertyAccessorMethod : FSharpGeneratedMethodBase, IFSharpGeneratedAccessor, IFSharpMember
```

Comment:
Why `Method` is needed? Accessor is always a method, isn't it?

Reply:
Resolve in C# explicitly separates accessors from methods

Reply:
> Resolve in C# explicitly separates accessors from methods.

What do you mean? `IAccessor` inherits `IMethod`.

Reply:
OK, this accessor is not an accessor, I see.